cms/static/js/views/pages/container.js

```diff
@@ -128,6 +128,12 @@ function($, _, Backbone, gettext, BasePage,
 
             }
 
+            window.addEventListener('message', (event) => {
+                if (event.data && event.data.type === 'refreshXBlock') {
+                    this.render();
+                }
+            });
+
```

Comment:
[optional]: If we have the ability to pass `isIframeEmbed` to a module, we should wrap all the necessary constructs in `isIframeEmbed`, including this message listener. What do you think?

```suggestion
            if (this.options.isIframeEmbed) {
                window.addEventListener('message', (event) => {
                    if (event.data && event.data.type === 'refreshXBlock') {
                        this.render();
                    }
                });
            }

```

Reply:
Fixed, thanks